### DISCUSSION TITLE
Toolbar for Windows Explorer

### DISCUSSION THREAD

**Discussion by qzmtch** [Open-Shell/Open-Shell-Menu#613] [Q&A] [answered]
I have such a question, is it possible to install only toolbar, without an open shell?

**Comment by bonzibudd** [comment, marked as answer]
Yes, you can uncheck the Start menu and IE in setup. Further configuration is done through Classic Explorer settings.

**Reply by qzmtch** [reply]
thanks, everything is set, ~~you can in future versions add color selection status bar? it's just that if you install various third-party themes, users usually install oldnewexplorer and after it, the big status bar is removed at the bottom through group policies, and I wanted to return it for myself through the open shell, but it can't change its color, it looks [like this](https://i.imgur.com/3qhFYZo.png) if in qttabbar you can still change the color, then in the status bar you cannot~~ sorry I wrote not there, I thought I was writing to developers I'll write to issues
